lms/static/sass/views/_homepage.scss

```diff
@@ -17,7 +17,7 @@ $learn-more-horizontal-position: calc(50% - 100px); // calculate the left positi
 
       .courses-listing-item {
         margin: 0 0 ($baseline*1.5) 0;
-        max-height: $course-card-height;
+        height: $course-card-height;
```

Comment:
I believe this line still needs to be reverted now that height is'n't being adjusted. (related to responsive cases)